certified-connectors/RecordedFutureNew/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1553 @@
+                              },
+                              "criticality":{
+                                "type":"integer",
+                                "format":"int32",
+                                "description":"Criticality",
+                                "x-ms-visibility":"internal"
+                              }
+                            }
+                          },
+                          "description":"Evidence details"
+                        },
+                        "riskString":{
+                          "type":"string",
+                          "description":"Risk string",
+                          "x-ms-visibility":"internal"
+                        },
+                        "rules":{
+                          "type":"integer",
+                          "format":"int32",
+                          "description":"Rules",
+                          "x-ms-visibility":"internal"
+                        },
+                        "criticality":{
+                          "type":"integer",
+                          "format":"int32",
+                          "description":"Criticality",
+                          "x-ms-visibility":"internal"
+                        },
+                        "riskSummary":{
+                          "type":"string",
+                          "description":"Recorded Future Risk Rules Summary",
+                          "title":"riskSummary",
+                          "x-ms-visibility":"advanced"
+                        }
+                      },
+                      "description":"Risk"
+                    },
+                    "links":{
+                      "$ref":"#/definitions/Links"
+                    }
+                  },
+                  "description":"Data"
+                }
+              }
+            }
+          }
+        },
+        "summary":"Vulnerability Enrichment",
+        "description":"Vulnerability Enrichment with Recorded Future data",
+        "parameters":[
+          {
+            "name":"id",
+            "in":"path",
+            "required":true,
+            "type":"string",
+            "x-ms-visibility":"important",
+            "description":"The Vulnerability ID (CVE, name) to lookup. Must be a single Vulnerability ID (CVE, name)",
+            "x-ms-summary":"Vulnerability ID (CVE, name) input",
+            "x-ms-url-encoding":"single"
+          },
+          {
+            "name":"fields",
+            "in":"query",
+            "required":true,
+            "type":"string",
+            "default":"intelCard,risk,links",
+            "x-ms-visibility":"internal"
+          }
+        ],
+        "operationId":"Vuln_E",
+        "x-ms-visibility":"advanced"
+      }
+    },
+    "/lookup/vulnerability/{id}/extension/{extension}":{
+      "get":{
+        "responses":{
+          "200":{
+            "description":"Default",
+            "schema":{
```

Comment:
@johannesh-recordedfuture Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
This endpoint has been removed.